docs/manual/warnings.md

```diff
@@ -41,6 +41,12 @@ You should be careful when changing images, environment variables, commands, or
 
 Other than the above, you can make any modifications to the pod definition you need to suit your requirements.
 
+## Limitations on FDB Port Customization
```

Comment:
How is "processNumber" going to be decided? Is the port going to be assigned to a process (or the additional processes) in a deterministic way?

Reply:
The process number is deterministic and is only used if more than one process is running per Pod e.g. when `storageServersPerPod` is set. The first process has the process number 1 and then the next processes will have the incremented process number e.g. 2, 3, 4 .., depending on how many processes should be running per Pod.

Reply:
So the process number gets assigned/decided based on the order that the processes start?

Reply:
Not really, the operator configures this in the fdbmonitor config.